test/cli.test.ts

```diff
@@ -294,6 +320,11 @@ describe('cli', () => {
       let stdout = ''
       ps.stdout?.on('data', (chunk) => (stdout += chunk.toString()))
       ps.stderr?.on('data', (chunk) => (stderr += chunk.toString()))
+      if (typeof timeoutClose === 'number') {
```

Comment:
Is this a correct solution?

Reply:
I think jest's timeout can't be used. If we don't kill the ps manually, watch mode will waiting at here

```ts
const code = await new Promise((resolve) => {
  ps.on('close', resolve)
})
```

test will always failed with timeout.